doc/source/ray-core/handling-dependencies.rst

```diff
@@ -367,10 +367,15 @@ The ``runtime_env`` is a Python dictionary or a Python class :class:`ray.runtime
   - Example: ``"pytorch_p36"``
 
 - ``env_vars`` (Dict[str, str]): Environment variables to set.  Environment variables already set on the cluster will still be visible to the Ray workers; so there is
-  no need to include ``os.environ`` or similar in the ``env_vars`` field.
+  no need to include ``os.environ`` or similar in the ``env_vars`` field.   
+  By default, these environment variables will override the same name environment variables on the cluster. 
+  If the user wants to append his value to the existing environment variable such as "PATH/LD_LIBRARY_PATH". he can refer to the example to set it.
```

Comment:
You can also reference the existing environment variable via `${ENV_VAR}` and achieve the append behavior (currently only PATH,LD_LIBRARY_PATH,DYLD_LIBRARY_PATH,LD_PRELOAD are supported). See below for an example.